expenses/migrations/0001_initial.py

```diff
@@ -11,7 +11,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('house', '0002_add_initial_data'),
+        ('house', '__first__'),
```

Comment:
What does this change do?
what does __first__ mean?
I have read here that it refers to the *first* migration. (and the change says it was 0002)
https://code.djangoproject.com/ticket/22861

Reply:
what the `__first__` means is that this migration needs to happen after the first migration of house.
because I removed the previous 0002 migration I noticed this expenses migration doesn't needs(depends) on 0002_add_initial_data(adds the cities and countries) but only needs the first house migration (0001_initial).
for readability and specifics I changed it to depend on `0001_initial`, but it works the same. hope this is more clear now :)

Reply:
It is, at least for me
Thanks!